components/02-molecules/page-title/_yds-page-title.scss

```diff
@@ -11,3 +11,14 @@
     max-width: var(--size-component-layout-width-highlight);
   }
 }
+
+.page-title__prefix {
+  @include tokens.h2-mallory-compact-book;
+
+  color: var(--color-basic-brown-gray);
+  text-transform: uppercase;
+}
+
+.prefix {
+  margin-top: var(--size-spacing-7);
+}
```

Comment:
@codeb-a Please move these lines into the `_yds-event-meta.scss` file and change the selector to match the update in the event-meta component, so it would look like this: 
```
.event-meta__past-event {
  margin-top: var(--size-spacing-7);
}
```